lightly/cli/config/get_config.py

```diff
@@ -6,4 +6,8 @@
 def get_lightly_config() -> DictConfig:
     config_path = Path(__file__).with_name("config.yaml")
     conf = OmegaConf.load(config_path)
+    # FIXME: remove this when hydra is completely dropped
```

Comment:
Nit: We usually put `TODO(name, date):`  for those kind of messages.

Reply:
Updated